### PR TITLE
activate translation in scheduled reports

### PR DESCRIPTION
**Pull request by sravfeyn**
http://manage.dimagi.com/default.asp?170534

### CODE REVIEW

**Comment by sravfeyn** [comment]
There is an opportunity for further improvement here. 
1. performance (see inline-comment)
2. In general all the celery tasks are out of scope of user language. So none of them are localized. May be we should audit all tasks that are user-facing for localization.

File: corehq/apps/reports/tasks.py [hunk 1]
```diff
@@ -121,11 +123,15 @@ def get_report_queue(report):
 @task(ignore_result=True)
 def send_report(notification_id):
     notification = ReportNotification.get(notification_id)
+    owner = WebUser.get(notification.owner_id)
```
Comment (sravfeyn) [inline corehq/apps/reports/tasks.py]: Is this going to result in a bad performance, thought about something like `get_cached_property`, but it doesn't make much sense here. Any other ideas?

**Comment by sravfeyn** [comment]
@esoergel code-buddy

File: corehq/apps/reports/tasks.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/reports/tasks.py]: I can't think of anything.

Comment (czue) [inline corehq/apps/reports/tasks.py]: seems ok since this runs in the background and the reports themselves are likely way slower.

**Comment by czue** [comment]
thanks!

**Comment by sravfeyn** [comment]
Should I audit user-facing tasks for localizations?

**Comment by czue** [comment]
I think that's a good backlog item, though not sure it's high priority or worth spending much time on now. I think reports + sms would be the big ones, and sms is managed separately.

Billing invoices might eventually be another one, but not sure those even have translations.

**Comment by sravfeyn** [comment]
Cool, should I add it to debt card on Trello?

**Comment by czue** [comment]
Not sure it's tech debt exactly. Maybe a P5+ feature in FB? I think it's
unlikely to happen explicitly unless you want to just take it on as a
personal initiative, in which case whatever approach/timing you want.

**Pull request merged**